test/controllers/lookups_controller_test.rb

```diff
@@ -126,16 +126,13 @@ def test_lookup_name_matching_only_glossary_term
   end
 
   def test_lookup_name_with_error
-    # Stub a method called by lookup_name in order to provoke an error.
-    LookupsController.any_instance.stubs(:fix_name_matches).
-      raises(RuntimeError)
     login
+    # Stub a method used inside `lookup_name` to provoke an error
```

Comment:
AFAIK, the point of this test was to cover a line in the code and have a reasonable assertion.
The new version does that, BUT the assertion is  looser than in the prior version.

Reply:
It's too convoluted for me to sort out what's going on.  If the coverage is the same, then I'm good with it.